### ISSUE TITLE
need to hit enter twice to send message on firefox

### ISSUE THREAD

**Issue by carlzoo** [glowing-bear/glowing-bear#706]
I am using firefox v43 b9. Whenever I switch to a different channel, I will need to hit enter twice to send the message. After a period of not typing, I will need to do this again to send the message. But after I hit enter twice, I only need to do it once, provided I don't not type for too long.

This is happening in private browsing with addons all disabled. Enable common readline keybindings in input bar is disabled.

**Comment by lorenzhs** [comment]
I can't reproduce this with Iceweasel 38.4.0 (from Debian unstable) nor 42.0 (from Debian experimental) so this sounds very much like a Firefox bug to me. Can you test Firefox 42.0? Also, which version of Glowing Bear are you using (www. or latest.?, i.e. `gh-pages` or `master` branch?)

**Comment by torhve** [comment]
Can you look in javascript console (Press F12) to see if there are any error messages generated?

**Comment by carlzoo** [comment]
@torhve there is no message related to this in the console
@lorenzhs i am running on gentoo, v42 is not in portage
this problem does not appear to happen in chrome, so maybe it is a firefox issue

**Comment by lorenzhs** [comment]
Right. I trust you don't have weird USE flags or the like (or do you use firefox-bin)? Have you tried firefox-bin 38.5.0? I'm a bit clueless right now so I wish you the best of luck figuring this out and please do make sure to let us know what you find out!

**Comment by lorenzhs** [comment]
I'm closing this for inactivity, if the issue persists with a stable release, please comment and I'll reopen.

**Comment by lorenzhs** [comment]
According to `benkilli1` on IRC, this issue still exists. Maybe they can fill in some more info?

**Comment by lorenzhs** [comment]
It might be related to old weechat versions - @carlzoo what's your weechat version? `benkilli1` was using 0.4.2 (very old)

**Comment by carlzoo** [comment]
At the time I believe I just used the version on glowing-bear.org, I cannot remember the exact number.

As of now I have firefox 45.2 installed using the current version on glowing-bear.org. No issues yet.

**Comment by lorenzhs** [comment]
I didn't mean the version of Glowing Bear, but rather the version of WeeChat installed on your server. Sorry if I wasn't clear. You can find that with the `/version` command.

**Comment by carlzoo** [comment]
The version is 1.3.

**Comment by lorenzhs** [comment]
Thanks. There goes the "ancient weechat" theory... ;)

**Comment by lorenzhs** [comment]
@carlzoo can you still reproduce this?
